### PR TITLE
word language model end-to-end example (AWD/RNNModel)

### PR DESCRIPTION
**Pull request by szha**

### CODE REVIEW

**Comment by szha** [comment]
@sxjscience

File: python/mxnet/gluon/model_zoo/text/lm.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        self.encoder = self._get_encoder()
+        self.decoder = self._get_decoder()
+
+    def _get_embedding(self):
+        embedding = nn.HybridSequential()
+        with embedding.name_scope():
+            embedding.add(nn.Embedding(len(self._in_vocab), self._embed_dim,
+                                       weight_initializer=init.Uniform(0.1)))
+            if self._dropout:
+                embedding.add(nn.Dropout(self._dropout))
```
Comment (cgraywang) [inline python/mxnet/gluon/model_zoo/text/lm.py]: nn.Dropout(self._dropout, axes = (0,))

Comment (szha) [reply python/mxnet/gluon/model_zoo/text/lm.py]: Doesn't seem needed. https://github.com/pytorch/examples/blob/master/word_language_model/model.py#L9

**Comment by szha** [comment]
sorry, this was merged prematurely. I've reverted the change on branch.
